core/src/movegen.c

```diff
@@ -1289,8 +1298,8 @@ void generate_moves(Generator *gen, Player *player, Rack *opp_rack,
     transpose(gen->board);
   }
 
-  // int do_bingo_gen = player->index == 1;  // DO NOT SUBMIT
-  int do_bingo_gen = true;  // player->index == 1;
+  int do_bingo_gen = player->index == 1;  // DO NOT SUBMIT
+  do_bingo_gen = true;  // player->index == 1;
```

Comment:
in the message for this commit when i said this was "valid", i had had do_bingo_gen == true for both players when what i meant was to compare with only one player using it. it is not actually valid yet. alarmingly, player 1 is outscoring player 0. possibly this is due to illegal moves because bingo_gen makes incorrect assumptions about how far left or right it can place moves?

autoplay 4000 1976 2009 15 2000 456.644000 61.152357 457.519250 60.912438

Reply:
this garbage
`   A B C D E F G H I J K L M N O      player_1                 ?EGMSTT   212
   ------------------------------  -> player_2                 ?IIIRST   219
 1|=     '       =       Q     = | --Tracking-----------------------------------
 2|  -       "       "   U   -   | AAAABCCDDEEEEEEFGHIIKLLMNOOOOOOOPPRRRRSTTUVW  44
 3|    -       '   '     E L     |
 4|'     -       '       N O   ' |
 5|        -           J A B     |
 6|  "       "       " I   I F   |
 7|    '       Z E A   V   ' A D |
 8|=     '       W H A E '     U |
 9|    '       '   '   Y A '   N |
10|  "       "       "   L   " N |
11|        -           X I     I |
12|'     -       '     U G     E |
13|    -       '   '     N -   S |
14|  -       "       Y R E N T   |
15|=     '       =       D     = |
   ------------------------------

play_move: 9D mIRITIS 73
   A B C D E F G H I J K L M N O   -> player_1                 ?EGMSTT   212
   ------------------------------     player_2                 EENOPSU   292
 1|=     '       =       Q     = | --Tracking-----------------------------------
 2|  -       "       "   U   -   | AAAABCCDDEEEEFGHIIKLLMOOOOOOPRRRRTTVW  37
 3|    -       '   '     E L     |
 4|'     -       '       N O   ' |
 5|        -           J A B     |
 6|  "       "       " I   I F   |
 7|    '       Z E A   V   ' A D |
 8|=     '       W H A E '     U |
 9|    ' m I R I T I S Y A '   N |
10|  "       "       "   L   " N |
11|        -           X I     I |
12|'     -       '     U G     E |
13|    -       '   '     N -   S |
14|  -       "       Y R E N T   |
15|=     '       =       D     = |
   ------------------------------
`

Reply:
the MIRITISYA bug is fixed, currently dealing with this miss

<img width="1450" alt="Screenshot 2023-12-07 at 12 53 46 AM" src="https://github.com/jvc56/MAGPIE/assets/57605/494cc395-0123-4ea1-b1ec-adad0e53a680">